packages/cinterop/build.gradle.kts

```diff
@@ -88,24 +88,53 @@ android {
     }
 }
 
-val nativeLibraryIncludesMacosUniversal = listOf(
-    "-include-binary", "$cinteropCorePath/capi_macos_universal/src/realm/object-store/c_api/librealm-ffi-static-dbg.a",
-    "-include-binary", "$cinteropCorePath/capi_macos_universal/src/realm/librealm-dbg.a",
-    "-include-binary", "$cinteropCorePath/capi_macos_universal/src/realm/parser/librealm-parser-dbg.a",
-    "-include-binary", "$cinteropCorePath/capi_macos_universal/src/realm/object-store/librealm-object-store-dbg.a"
+val nativeLibraryIncludesMacosUniversalRelease = listOf(
```

Comment:
Extremely verbose. I guess there are some veriations, but I guess we could maybe define the libs 
```
val libs = setOf("lib/librealm-ffi-static.a"....)
val libsDbg = setOf("lib/librealm-ffi-static.a"....)
```
And define the compiler arguments by something from these, by some mapping
```
val nativeLibraryIncludesIosArm64Debug = libsDbg.map { ... }
```